deisctl/deisctl.go

```diff
@@ -32,6 +32,7 @@ Commands, use "deisctl help <command>" to learn more:
   install           install components, or the entire platform
   uninstall         uninstall components
   list              list installed components
+  list-machines     Enumerate the current hosts in the cluster
```

Comment:
I'd argue for `nodes` as a better choice in prep for v2 (would be handy to make `deisctl nodes` be equivalent to `kubectl get nodes`)